{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2019-6690",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://lists.opensuse.org/opensuse-security-announce/2019-02/msg00008.html",
          "name": "SU-2019:0143-1",
          "refsource": "SUSE",
          "tags": [

          ]
        },
        {
          "url": "http://lists.opensuse.org/opensuse-security-announce/2019-02/msg00058.html",
          "name": "SUSE-SU-2019:0239-1",
          "refsource": "SUSE",
          "tags": [

          ]
        },
        {
          "url": "http://packetstormsecurity.com/files/151341/Python-GnuPG-0.4.3-Improper-Input-Validation.html",
          "name": "http://packetstormsecurity.com/files/151341/Python-GnuPG-0.4.3-Improper-Input-Validation.html",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "http://www.securityfocus.com/bid/106756",
          "name": "106756",
          "refsource": "BID",
          "tags": [

          ]
        },
        {
          "url": "https://lists.debian.org/debian-lts-announce/2019/02/msg00021.html",
          "name": "[SECURITY] [DLA 1675-1] 20190214 python-gnupg security update",
          "refsource": "MLIST",
          "tags": [

          ]
        },
        {
          "url": "https://pypi.org/project/python-gnupg/#history",
          "name": "https://pypi.org/project/python-gnupg/#history",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://seclists.org/bugtraq/2019/Jan/41",
          "name": "20190125 CVE-2019-6690: Improper Input Validation in python-gnupg",
          "refsource": "BUGTRAQ",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "python-gnupg 0.4.3 allows context-dependent attackers to trick gnupg to decrypt other ciphertext than intended. To perform the attack, the passphrase to gnupg must be controlled by the adversary and the ciphertext should be trusted. Related to a \"CWE-20: Improper Input Validation\" issue affecting the affect functionality component."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2019-03-21T16:01Z",
  "lastModifiedDate": "2019-03-21T16:01Z"
}